contents/tutorials/github-star-tracker.md

```diff
@@ -0,0 +1,51 @@
+---
+title: How to track GitHub stars in PostHog
+date: 2024-03-25
+author: ["ian-vanagas"]
+tags:
+  - events
+---
+
+GitHub stars are a popular way to track the success of open-sourced projects. Beyond seeing how many stars a repo has, GitHub does little to help you track or analyze these stars. 
+
+To help you make better use of stars, this tutorial shows you how capture them using a combination of PostHog, GitHub webhooks, and [Val Town](https://val.town/), platform for writing, running, and scheduling JavaScript functions in your browser.
+
+## Creating a webhook handler in Val Town
+
+Our webhook handler will simply accept a `POST` request, get the data we want from that request, and capture a PostHog event with that data. Val Town is great for doing this because we can reuse the `capturePostHogEvent` we created earlier and you can fork, modify, and run [the code I already wrote here](https://www.val.town/v/ianvph/posthogGitHubStarCapture):
```

Comment:
```suggestion
Our webhook handler will accept a `POST` request and capture a PostHog event with the data in the request. Val Town is great for doing this because you can reuse [the code I already wrote here](https://www.val.town/v/ianvph/posthogGitHubStarCapture):
```

Reply:
Its not clear to me how to reuse this code in a val town?